src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/jql/Jql.kt

```diff
@@ -0,0 +1,19 @@
+package com.atlassian.performance.tools.jiraactions.api.jql
+
+import com.atlassian.performance.tools.jiraactions.api.SeededRandom
+import com.atlassian.performance.tools.jiraactions.api.page.IssuePage
+
+interface JqlContext {
+    fun seededRandom(): SeededRandom
+    fun issuePage(): IssuePage?
+}
+
+interface JqlSupplier {
```

Comment:
Is it possible to reduce the number of public interfaces? It seems that our API grows quickly.

Reply:
supplier is replacement for PrescriptionType which is public and coukld be removed only witch major change